integration/release_upgrade_linux_test.go

```diff
@@ -486,9 +514,15 @@ done
 func newPodTCtx(t *testing.T, rSvc cri.RuntimeService,
 	name, ns string, opts ...PodSandboxOpts) *podTCtx {
 
-	t.Logf("Run a sandbox %s in namespace %s", name, ns)
+	return newPodTCtxWithRuntimeHandler(t, rSvc, name, ns, "", opts...)
+}
+
+func newPodTCtxWithRuntimeHandler(t *testing.T, rSvc cri.RuntimeService,
```

Comment:
If there is a structure similar to the one below, I think this part of the code is easier to read and maintain.

```
type ctrdConfig struct {
	t       *testing.T
	workdir string
	version     int
}
```

Reply:
This new method is to allow us to run pod with different runtime. I think it's not related to runtime config you mentioned. Yeah, I think we should use function options here. I just don't get that why runtime handler is not that part of pod config. We can improve that in follow up